packages/ui/src/components/DetailView/DetailView.tsx

```diff
@@ -10,17 +10,24 @@ import { getTokenValue, withStaticProperties } from 'tamagui';
 import { Text, View, YStack } from '../../core';
 import { useStickyUnread } from '../../hooks/useStickyUnread';
 import AuthorRow from '../AuthorRow';
+import { BigInput } from '../BigInput';
 import Scroller from '../Channel/Scroller';
 import { ChatMessage } from '../ChatMessage';
 import { MessageInput } from '../MessageInput';
 import { DEFAULT_MESSAGE_INPUT_HEIGHT } from '../MessageInput/index.native';
 
 export interface DetailViewProps {
   post: db.Post;
+  channelType?: db.ChannelType;
```

Comment:
Can we avoid passing channelType around by just pulling type from channel id?

Reply:
Good point